### PR TITLE
Add locale name from language definition

### PR DESCRIPTION
**Pull request by danny-may**
Language names should be in the language its describing rather than english

### CODE REVIEW

File: src/crowdin/index.ts [hunk 1]
```diff
@@ -67,7 +67,11 @@ export class CrowdinTranslationSource implements ITranslationSource {
             this.#languageKeys.delete(language.locale);
             this.#languageData.delete(language.locale);
         } else {
-            this.#languageKeys.set(language.locale, { name: language.name, keys: new Set(lookup.keys()) });
+            this.#languageKeys.set(language.locale, {
+                name: lookup.get('config.localeName') ?? language.name,
+                keys: new Set(lookup.keys())
+            });
+            lookup.delete('config.localeName');
```
Comment (HunteRoi) [inline src/crowdin/index.ts:74]: why do you delete it?

Comment (danny-may) [reply src/crowdin/index.ts:74]: Just so it doesnt contribute to any stats, this is the only place its used

Comment (HunteRoi) [reply src/crowdin/index.ts:74]: what's the goal of the lookup object actually?

Comment (danny-may) [reply src/crowdin/index.ts:74]: It contains the mapping of keys to translated templates, so the english templates can be swapped out

**Review by HunteRoi — APPROVED** [review]

**Pull request merged**